spec/descriptions/getEvents.md

```diff
@@ -8,6 +8,8 @@ This endpoint retrieves all available events for the requested timeframe.
 - **excludeTriggeredBefore:** Whether to exclude events that have been triggered before the requested timeframe in order to enable searching for events that have started within the given timeframe, excluding events that are previously active already. This is useful for 3rd party integrations that fetch events from Instana with a scheduled batch job in a fixed interval using tumbling windows, when you only care about new events.
 This option is more restrictive than `filterEventUpdates` and does not inform about event state updates that got `CLOSED` in the timeframe of the query if not also the start time of the event is within that query timeframe.
 - **filterEventUpdates:** Filters results to event updates only. This means that an event is only included when its event state changed in the given query timeframe. This is useful for 3rd party integrations that fetch events from Instana with a scheduled batch job in a fixed interval using a tumbling windows, when you care about event state updates.
+- **includeAgentMonitoringIssues:** Default to false, which excludes Agent Monitoring Issues. Set to true to include those events in the result.
+- **includeKubernetesInfoEvents:** Default to false, which excludes Kubernetes Info Events. Set to true to include those events in the result.
```

Comment:
**Optional suggestion:**
See the example from another section in the docs below:
<img width="789" alt="Screenshot 2023-05-22 at 11 38 01 AM" src="https://github.com/instana/openapi/assets/2537736/e31cf495-ecde-43ce-b87f-0ecacc687da0">

Values such as `true` or `false` could be highlighted as code. And in other descriptions, we mention the default value last, instead of first. Also, a minor thing that could be improved is that you kind of explain the flag the opposite way round. Such as the flag `includeXXX`, that it "excludes" things. Might be more naturally to explain the flag in the way it is names 🙂 

To follow that, we could change it to the following:

> - **includeAgentMonitoringIssues:** Optional flag to indicate whether to include Agent Monitoring Issues. The default is `false`.
> - **includeKubernetesInfoEvents:** Optional flag to indicate whether to include  Kubernetes Info Events. The default is `false`.

Reply:
That is more clear. Thank you for the suggestions.